Agents/LSOAInputAgent/README.md

```diff
@@ -0,0 +1,183 @@
+# Description
+
+The `LSOAInput agent` is dedicated to process data around the UK Lower-layer Super Output Area (LSOA), and wrap the data as triples based on ontologies, and upload the triples into the knowledge graph.
+
+Currently this agent is focusing on the data of electricity consumption, gas consumption, fuel poverty, climate (temperature) and geometric shape to perform the use case of analysing the deployment of heat pump. The data instatiated in the knowledge graph follows[Ontoclimate](http://www.theworldavatar.com/ontology/ontogasgrid/ontoclimate.owl), [Ontofuelpoverty](http://www.theworldavatar.com/ontology/ontofuelpoverty/ontofuelpoverty.owl) and [Ontogasgrid](https://github.com/cambridge-cares/TheWorldAvatar/tree/main/GasGrid) ontologies in the [TheWorldAvatar](https://github.com/cambridge-cares/TheWorldAvatar). 
```

Comment:
What do you mean, "Currently this agent is focusing on..."?

Can you please provide a better description of what the agent does?